currency.js

```diff
@@ -1,37 +1,68 @@
-var request = require('request');
+const request = require('request');
 
-exports.run = function(api, event) {
-    var query = event.body.substr(10),
-        parts = query.split(' ');
+const supportedCurrencies = ['EUR',"AUD","BGN","BRL","CAD","CHF","CNY","CZK","DKK","GBP","HKD","HRK","HUF","IDR","ILS","INR","JPY","KRW","MXN","MYR","NOK","NZD","PHP","PLN","RON","RUB","SEK","SGD","THB","TRY","USD","ZAR"];
 
-    //If we don't have the right number of parts, give up
-    if (parts.length !== 4) {
-        api.sendMessage("That looks wrong. You should try harder.", event.thread_id);
+const number = '(-?[0-9](,|[0-9])+(.[0-9]+)?) ?';
+const currencies = `(${supportedCurrencies.join('|')})`;
+const regex = `${number} ?${currencies}( in ${currencies})?`;
+const re = new RegExp(regex, 'gi');
+
+let conversions = null;
+
+exports.match = (event, commandPrefix) => {
```

Comment:
Is this match function strictly nessecary? Would the kassy.json command be simpler?